src/librustc/middle/lang_items.rs

```diff
@@ -90,6 +90,10 @@ impl LanguageItems {
         }
     }
 
+    pub fn require_owned_box(&self) -> Result<ast::DefId, String> {
```

Comment:
That's right, but there's not a great way to handle the error unless you get the error message which comes with the `require` version. I guess you could make up your own message. That's why I use "'right'" rather than "right" - it's not too important. But it is an error if its not there, so Result feels right.